### DISCUSSION TITLE
Add related blocklist for safe search and parental control AGH...

### DISCUSSION THREAD

**Discussion by t0ny54** [AdguardTeam/AdGuardHome#4229] [General]
Add related blocklist for safe search and parental control AGH...

Is it possible when I choice safe search or parental control, add blocking unsafe search for safe search option and block bypass method for parental control?

Use safe search and blocking unsafe search...
Source:
https://raw.githubusercontent.com/nextdns/metadata/master/parentalcontrol/safesearch-not-supported

Use AdGuard parental control web service, blocking bypass-methods and proxy
Source:
https://raw.githubusercontent.com/nextdns/metadata/master/parentalcontrol/bypass-methods
https://raw.githubusercontent.com/nextdns/piracy-blocklists/master/proxies

Reason:
User/Client can bypass safe search and bypass adult material with doh, dot, etc...
May be i can add manually in dns blocklist, but i like AGH add that list without add manually...

Thank You

**Comment by ainar-g** [comment]
>  User/Client can bypass safe search and bypass adult material with doh, dot, etc...

As long as you have set up your clients and your AdGuard Home correctly, this shouldn't really happen.  To forward clients' IP addresses to an AGH instance behind an HTTP proxy, use the `trusted_proxies` configuration parameter.
